### PR TITLE
[Google Search] Improve extension functionality

### PR DESCRIPTION
**Pull request by mblode**
## Description

- Add an action for copy suggestion to clipboard #2664
- Add a toggle for search history #2663
- Always have current search at top #887
- Improve searching of history #677

## Screencast

https://www.loom.com/share/34dc0a86555e41de80b88c513dc7868a

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/google-search/src/index.tsx [hunk 1]
```diff
@@ -38,6 +31,7 @@ export default function Command() {
                   />
 
                   <CopyToClipboardAction title="Copy URL to Clipboard" content={item.url} />
+                  <CopyToClipboardAction title="Copy query to Clipboard" content={item.query} />
```
Comment (pernielsentikaer) [inline extensions/google-search/src/index.tsx:34]: Shouldn't it be the suggestion and not the query?

File: extensions/google-search/package.json [hunk 2]
```diff
@@ -14,24 +17,37 @@
       "mode": "view"
     }
   ],
+  "preferences": [
+    {
+      "name": "rememberSearchHistory",
+      "type": "checkbox",
+      "required": false,
+      "default": true,
+      "label": "Remember search history?",
```
Comment (pernielsentikaer) [inline extensions/google-search/package.json:26]:
```suggestion
      "label": "Remember Search History",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your update 💪 

I have now tested your extension, and I have some suggestions ready for you 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/google-search/src/index.tsx (hunk 1 above)
Comment (mblode) [reply extensions/google-search/src/index.tsx]: Good spot! That's updated

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**